benchmarks/Microsoft.AspNetCore.Routing.Performance/Matchers/FastPathTokenizerSmallBenchmark.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using BenchmarkDotNet.Attributes;
+
+namespace Microsoft.AspNetCore.Routing.Matchers
+{
+    public class FastPathTokenizerSmallBenchmark
+    {
+        private const int MaxCount = 32;
+        private static readonly string Input = "/hello/world/cool";
+
+        // This is a naive reference implementation. We expect to do better.
+        [Benchmark(Baseline = true)]
+        public unsafe void Baseline()
+        {
+            var path = Input;
+            var segments = stackalloc PathSegment[MaxCount];
+
+            int count = 0;
```

Comment:
This is used in 2 benchmark classes. Move it somewhere common?

Reply:
Sure